### PR TITLE
[Avito] Update Avito data with 1 link pred task, 1 regression task and 2 binary classification task

### PR DESCRIPTION
**Pull request by XinweiHe**
Follow up of https://github.com/snap-stanford/relbench/pull/171.

Summary:
1. Downsample the avito dataset to 100k in target table from 500k for faster iteration.
2. Propose a link prediction task to: Predict the list of ads user will visit in the next 4 days.
3. Propose a regression task to: Assuming the ad will be clicked in the next 4 days, predict the CTR for each ad.
4. Propose a binary classification task to: Predict whether each customer will visit more than one ad in the next 4 days.
5. Propose a binary classification task to: Predict the each customer will click on more than one ads in the next 4 day.

Benchmarking Result:
Screenshot 2024-06-13 at 1 51 50 PM

Screenshot 2024-06-13 at 1 23 20 PM

Screenshot 2024-06-13 at 1 23 45 PM

Screenshot 2024-06-13 at 11 12 32 PM

### CODE REVIEW

File: relbench/tasks/avito.py [hunk 1]
```diff
@@ -14,21 +17,125 @@
 )
 
 
+class AdsClicksTask(RelBenchNodeTask):
+    r"""Assuming the ad will be clicked in the next 4 days, predict the CTR
+    for each ad.
```
Comment (rusty1s) [inline relbench/tasks/avito.py:22]: Click-Through-Rate (CTR)

File: relbench/tasks/avito.py [hunk 2]
```diff
@@ -14,21 +17,125 @@
...
+            df=df,
+            fkey_col_to_pkey_table={"UserID": "entity_table"},
+            pkey_col=None,
+            time_col="timestamp",
+        )
+
+
 class UserClicksTask(RelBenchNodeTask):
-    r"""Predict the total number of ads each customer will click in the
+    r"""Predict the each customer will click on more than one ads in the
```
Comment (rusty1s) [inline relbench/tasks/avito.py:124]:
```suggestion
    r"""Predict whether each customer will click on more than one ads in the
```

Comment (rusty1s) [inline relbench/tasks/avito.py:124]: What's the difference between visit and click?

File: relbench/tasks/avito.py [hunk 3]
```diff
@@ -66,10 +179,10 @@ def make_table(self, db: Database, timestamps: "pd.Series[pd.Timestamp]") -> Tab
         )
 
 
-class UserAdClickTask(RelBenchLinkTask):
-    r"""Predict the list of ads user will click in the next 4 days"""
+class UserAdVisitTask(RelBenchLinkTask):
+    r"""Predict the list of ads user will visit in the next 4 days"""
```
Comment (rusty1s) [inline relbench/tasks/avito.py:183]:
```suggestion
    r"""Predict the list of ads a user will visit in the next 4 days"""
```

**Review by rusty1s — APPROVED** [review]

File: relbench/tasks/avito.py (hunk 2 above)
Comment (XinweiHe) [reply relbench/tasks/avito.py:124]: click signal is in SearchStream table, while visit signal is in VisitsStream table.

Comment (rusty1s) [reply relbench/tasks/avito.py]: Ok, but is that a completely different task? Are they similar? Do we need both?

Comment (XinweiHe) [reply relbench/tasks/avito.py]: Yes, they are completely different tasks.

**Pull request merged**